akka-docs/src/main/paradox/additional/monitoring.md

```diff
@@ -0,0 +1,3 @@
+# Monitoring
+
+Aside from log monitoring and the monitoring provided by your platform provider, the [Lightbend Telemetry](https://developer.lightbend.com/docs/telemetry/current/instrumentations/akka/akka.html) product (which is available through a [Lightbend Platform Subscription](https://www.lightbend.com/lightbend-platform-subscription)) can provide additional insights in the run-time characteristics of your application.
```

Comment:
That is one short page, but sure

Reply:
To give a short summary of what Lightbend Telemetry we could copy the same as mentioned elsewhere: Distributed tracing, metrics and monitoring for Akka Actors, Cluster, HTTP and more.